src/vmspawn/vmspawn-util.c

```diff
@@ -191,7 +191,13 @@ int find_ovmf_config(int search_sb, OvmfConfig **ret) {
                         continue;
                 }
 
-                int sb_present = !!strv_find(fwd->features, "secure-boot");
+                int enrolled_keys = strv_contains(fwd->features, "enrolled-keys");
+                if (enrolled_keys) {
```

Comment:
Please call strv_contains() in `if`, then `enrolled_keys` can be dropped.

Reply:
Fixed in 773ca1def8.

Reply:
Fixed in 773ca1def8.